pkg/apis/componentconfig/types.go

```diff
@@ -503,6 +503,8 @@ type KubeControllerManagerConfiguration struct {
 	// horizontalPodAutoscalerSyncPeriod is the period for syncing the number of
 	// pods in horizontal pod autoscaler.
 	HorizontalPodAutoscalerSyncPeriod unversioned.Duration `json:"horizontalPodAutoscalerSyncPeriod"`
+	// URL to an external heapster backend
+	Heapster string `json:"heapster"`
```

Comment:
I agree we should start removing direct references to Heapster as we have metric api coming.

Having `url` in the name is good. How about `MetricAPIUrl` or `MetricSourceUrl`?